src/js/media/html5.js

```diff
@@ -61,6 +61,9 @@ vjs.Html5 = vjs.MediaTechController.extend({
 
     if (this['featuresNativeTextTracks']) {
       this.on('loadstart', vjs.bind(this, this.hideCaptions));
+    } else {
+      // allow us to view the in-band metadata tracks from our shimmed TextTrackList
+      this.on('loadedmetadata', vjs.bind(this, this.addInBandMetadataTracks));
```

Comment:
I'm not sure that inband tracks will show up by `loadedmetadata`. Did you find this in the spec or discover it experimentally?